lua/telescope/builtin/init.lua

```diff
@@ -483,7 +483,7 @@ builtin.lsp_dynamic_workspace_symbols = require_on_exported_call("telescope.buil
 ---@field root_dir string|boolean: if set to string, get diagnostics only for buffers under this dir otherwise cwd
 ---@field no_unlisted boolean: if true, get diagnostics only for listed buffers
 ---@field no_sign boolean: hide DiagnosticSigns from Results (default: false)
----@field line_width number: set length of diagnostic entry text in Results
+---@field line_width string|number: Set length of diagnostic entry text in Results. Use 'full' for full untruncated text
```

Comment:
super nitpicking but for the sake on consistency, can we lowercase the "Set length..."?